src/index.html

```diff
@@ -1,13 +1,46 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no,
+                   initial-scale=1.0, maximum-scale=1.0,
+                   minimum-scale=1.0">
```

Comment:
Unfortunately, its not correct formatting. Each attribute should be on a new line and `<` `>` should be on the same distance

```
<meta
  name="viewport"
  content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
>
```